### DISCUSSION TITLE
익스텐션 관련 추가 API 논의

### DISCUSSION THREAD

**Discussion by dmdgpdi** [Kernel360/F2-BAGUNI#190] [메모장]
현재는 픽(북마크)만 가능한데 픽 했던 url로 접근 시에 익스텐션을 열면 다시 픽(북마크)을 해야하는 화면이 보입니다.
이는 클라이언트에서 해당 url을 북마크 했는지 알 수 없기에, url를 query에 담아서 보내 해당 url을 픽했는지 확인할 수 있어야 한다고 생각하는데 다른 분들의 의견이 궁금합니다.
또한 픽을 했을 때의 히스토리(최근 픽 10개)를 볼 수 있으면 좋을 것 같은 의견도 있는데(수형님) 이것도 논의를 해보면 좋을 것 같습니다!

**Comment by Gyaak** [comment]
@dmdgpdi
v2에서는 url로 픽 조회 api 추가하도록 할게요

익스텐션에서 태그crud, 픽 추가 이외에 가장 최근에 등록한 픽 목록(?)을 보여주면 좋을거같은데
다들 @kimminkyeu @sangwonsheep @enigsuss 의견 부탁드려요

**Comment by kimminkyeu** [comment]
네 url로 이미 픽했는지 체크하는 api는 v2에 추가하겠습니다.
최근 픽 리스트는 query param으로 날짜 범위별로 조회할 수 있게 만들면 프론트에서 여기저기 잘 써먹을 것 같아요
ex.
- API
```
GET api/pick?date_begin={?}&date_end={?}
```

**Comment by sangwonsheep** [comment]
query param으로 날짜 범위별로 조회하는 api 추가하도록 하겠습니다.
생성일 기준으로 하면 될 것 같다는 생각이 드네요.
